tests/ci/ci.test

```diff
@@ -2,5 +2,5 @@ control
 thr mpi dcp rst bit dbg
 regional_control
 thr dcp
-cpld_control_c96_p8
+cpld_control_c96_noaero_p8
```

Comment:
Developers are looking into the issue with cpld_control_c96_p8 (issue #1103), maybe we can wait until that is fixed. Since the control test is using GOCART, I'd suggest not changing the test case.